app/uk/gov/hmrc/cardpaymentfrontend/utils/OriginExtraInfo.scala

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cardpaymentfrontend.utils
+
+import uk.gov.hmrc.cardpaymentfrontend.models.extendedorigins.{DefaultExtendedOrigin, ExtendedOrigin, ExtendedPfSa, ExtendedPfVat}
+import payapi.corcommon.model.{Origin, Origins}
+
+import javax.inject.{Inject, Singleton}
+trait PaymentMethod
+
+final case class Card() extends PaymentMethod
```

Comment:
These could be case objects since there's no info in parameter list. Doesn't really matter too much though